compiler/docgen.nim

```diff
@@ -1413,14 +1420,10 @@ proc commandDoc*(cache: IdentCache, conf: ConfigRef) =
 proc commandRstAux(cache: IdentCache, conf: ConfigRef;
                    filename: AbsoluteFile, outExt: string) =
   var filen = addFileExt(filename, "txt")
-  var d = newDocumentor(filen, cache, conf, outExt)
-
-  d.isPureRst = true
+  var d = newDocumentor(filen, cache, conf, outExt, isPureRst=true)
```

Comment:
```suggestion
  let d = newDocumentor(filen, cache, conf, outExt, isPureRst=true)
```

Reply:
`let` does not work here because `d` is used as `var` in `finishGenerateDoc`.
I feel I will sometime go through the file and convert some `var` to `let` to satisfy your let-elimination urge :-)